packages/react/src/drafts/ActionBar/ActionBar.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import {behavesAsComponent} from '../../utils/testing'
+import {render as HTMLRender} from '@testing-library/react'
+import {axe} from 'jest-axe'
+
+import ActionBar from './'
+import {BoldIcon, CodeIcon, ItalicIcon, LinkIcon} from '@primer/octicons-react'
+
+const SimpleActionBar = () => (
+  <ActionBar>
+    <ActionBar.IconButton icon={BoldIcon} aria-label="Default"></ActionBar.IconButton>
+    <ActionBar.IconButton icon={ItalicIcon} aria-label="Default"></ActionBar.IconButton>
+    <ActionBar.Divider />
+    <ActionBar.IconButton icon={CodeIcon} aria-label="Default"></ActionBar.IconButton>
+    <ActionBar.IconButton icon={LinkIcon} aria-label="Default"></ActionBar.IconButton>
+  </ActionBar>
+)
+
+describe('ActionBar', () => {
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  behavesAsComponent({
+    Component: ActionBar,
+    options: {skipAs: true, skipSx: true},
+    toRender: () => <SimpleActionBar />,
+  })
+
+  it('should have no axe violations', async () => {
+    const {container} = HTMLRender(<SimpleActionBar />)
+    const results = await axe(container)
+    expect(results).toHaveNoViolations()
+  })
```

Comment:
Just wanted to share that axe is also available in the e2e tests but with the additional github rules if that's of interest! Wasn't sure how obvious it was.